airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/avro_parser.py

```diff
@@ -157,5 +157,11 @@ def _to_output_value(avro_format: AvroFormat, record_type: Mapping[str, Any], re
             return record_value
         if record_type.get("logicalType") == "uuid":
             return uuid.UUID(bytes=record_value)
+        elif record_type.get("logicalType") == "decimal":
+            return str(record_value)
```

Comment:
[decimals should always be converted to string](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/avro_parser.py#L107-L117)